04LessonFour/in-class-project/css/style.css

```diff
@@ -47,6 +48,16 @@ a  {
    margin-right: 500px;
 }
 
+.processor1 {
+   position: absolute;
+   top:0;
+   padding-top: 40px;
+   left: 40%;
+   width:100%;

```

Comment:
this width of 100% makes the box be really big! remember, %s are relative to their parent. so .processor1's parent's width is taken into account here. if you need help on this, let me know!